pkg/ccl/importccl/import_stmt.go

```diff
@@ -238,7 +259,14 @@ func importPlanHook(
 	if !ok {
 		return nil, nil, nil, false, nil
 	}
-	telemetry.Count("import.total.attempted")
+
+	format := roachpb.IOFileFormat{}
+	var fileFormat roachpb.IOFileFormat_FileFormat
+	if fileFormat, ok = resolveImportFileFormat[importStmt.FileFormat]; !ok {
+		return nil, nil, nil, false, unimplemented.Newf("import.format", "unsupported import format: %q", importStmt.FileFormat)
+	}
+	format.Format = fileFormat
+	addToFileFormatTelemetry(fileFormat.String(), "attempted")
```

Comment:
I'm not sure we need any of these optimizations (re keeping track of which formats we have seen and when).
I would just use addToFileFormatTelemetry() directly.  I don't think you should worry about unsupported format as you do above.  This is enforced via sql parsing.

Reply:
done.